### DISCUSSION TITLE
Convert ARC parameters to the point list format required by LWPOLYLINE

### DISCUSSION THREAD

**Discussion by zhuochengs** [mozman/ezdxf#875] [General]
Hi, I have a requirement to have a set of Lines and Arcs that are determined to be a closed curve. I want to use this library to turn them into a closed polyline. What should I do? I don't want to open the dwg file in CAD to handle this matter, thank you.

Line is possible, but how can Arc be converted into a (x, y, start_width, end_width, bulk) parameter?

**Comment by mozman** [comment]
https://ezdxf.mozman.at/docs/math/core.html#ezdxf.math.arc_to_bulge

**Reply by zhuochengs** [reply]
Thanks very much!
But how could I use the return values (start_point, end_point, bulge) to use In the LWPOLYLINE.add_lwpolyline() the points parameter?
I mean how to use it ? Can you give me  an  exampes? thanks~
